CHANGELOG.rst

```diff
@@ -4,7 +4,7 @@ CHANGELOG
 This document describes changes between each past release as well as
 the version control of each dependency.
 
-0.10.0 (unreleased)
+0.10.0 (2016-10-28)
```

Comment:
We will take this opportunity to move on to version 1.0 with no more patch version as decided previously with OPS.